src/routes/consent.ts

```diff
@@ -10,7 +10,38 @@ import {
 } from "@ory/client"
 import { UserConsentCard } from "@ory/elements-markup"
 import bodyParser from "body-parser"
-import csrf from "csurf"
+import { doubleCsrf } from "csrf-csrf"
+import { Request, Response, NextFunction } from "express"
+
+// Sets up csrf protection
+const {
+  generateToken, // Use this in your routes to provide a CSRF hash + token cookie and token.
+  invalidCsrfTokenError,
+  doubleCsrfProtection, // This is the default CSRF protection middleware.
+} = doubleCsrf({
+  getSecret: () => "VERY_SECRET_VALUE", // A function that optionally takes the request and returns a secret
```

Comment:
We need to set a secret in this library for the csrf hash generation.

Reply:
I guess this should be an environment variable which we will also need to control in AX.